recipes-wpe/wpebackend-rdk/wpebackend-rdk.inc

```diff
@@ -25,15 +25,16 @@ def backendselector(d):
     compositor = bb.utils.contains('DISTRO_FEATURES', 'compositor', True, False, d)
 
     if compositor == False and wayland == False:
-        return 'rpi'
+        return 'platform'
```

Comment:
Can we have this fix as a separate commit? Just not to mix fixes with refactoring

Reply:
done